Time filter fails for certain values

The following error is observed when one of the parameters is `Instant.MIN`

```
Caused by: java.lang.ArithmeticException: long overflow
    at java.lang.Math.multiplyExact(Math.java:892)
    at java.time.Instant.toEpochMilli(Instant.java:1237)
    at com.microsoft.dhalion.core.TableUtils.filterTime(TableUtils.java:42)
    at com.microsoft.dhalion.core.OutcomeTable.filterTime(OutcomeTable.java:110)
    at com.microsoft.dhalion.core.SymptomsTable.between(SymptomsTable.java:109)
```

--- a/pom.xml
+++ b/pom.xml
@@ -5,7 +5,7 @@
 
   <groupId>com.microsoft.dhalion</groupId>
   <artifactId>dhalion</artifactId>
-  <version>0.2.2</version>
+  <version>0.2.3</version>
   <packaging>jar</packaging>
 
   <name>Dhalion</name>

--- a/src/main/java/com/microsoft/dhalion/policy/PoliciesExecutor.java
+++ b/src/main/java/com/microsoft/dhalion/policy/PoliciesExecutor.java
@@ -120,7 +120,6 @@ public class PoliciesExecutor {
     this.executor.shutdownNow();
   }
 
-
   public static class ExecutionContext {
     private final MeasurementsTable.Builder measurementsTableBuilder;
     private final SymptomsTable.Builder symptomsTableBuilder;
@@ -139,7 +138,7 @@ public class PoliciesExecutor {
     }
 
     private void captureCheckpoint() {
-      previousCheckpoint = checkpoint != null ? checkpoint : Instant.MIN;
+      previousCheckpoint = checkpoint != null ? checkpoint : Instant.EPOCH;
       checkpoint = policy.getNextCheckpoint();
     }
 

